runtime/src/iree/hal/drivers/vulkan/api.cc

```diff
@@ -15,8 +15,6 @@
 #include "iree/hal/drivers/vulkan/dynamic_symbols.h"
 #include "iree/hal/drivers/vulkan/util/ref_ptr.h"
 
-using namespace iree::hal::vulkan;
```

Comment:
We do this in a bunch of places. Should we update those too?

https://github.com/search?q=%22using+namespace+iree%3A%3Ahal%3A%3Avulkan%3B%22+repo%3Aopenxla%2Firee&type=code

Reply:
We had ideas to rewrite the entire Vulkan HAL in C instead of C++. I just noticed while editing this file that some types were fully qualified and some took advantage of the `using namespace`, so I standardized them on fully qualified.